bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableConfiguration.java

```diff
@@ -23,16 +23,16 @@
 import java.util.concurrent.TimeUnit;
 import java.util.Map.Entry;
 
+import org.apache.beam.repackaged.beam_sdks_java_io_hbase.com.google.common.base.Strings;
```

Comment:
What is so special about Strings?

Reply:
I missed a commit before I pushed.  I removed `org.apache.beam.repackaged`.  PTAL.